src/pages/user/[username]/index.tsx

```diff
@@ -81,6 +82,8 @@ function ProfileOverview() {
 
   return (
     <>
+      <Meta title={`User Profile | ${user.username}`} description="View user profile and content" />
```

Comment:
I think this is already handled inside the `ProfileLayout`, please double check this is not overwriting